dashboards/Experimental/Patroni_Details.json

```
@@ -0,0 +1,3766 @@
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 5,
+        "tagValuesQuery": "",
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "current": {},
+        "datasource": "Metrics",
+        "definition": "label_values({__name__=~\"pg_up|mysql_up|mongodb_up|proxysql_mysql_status_active_transactions\"}, schema)",
+        "hide": 2,
+        "includeAll": true,
+        "label": "Schema",
+        "multi": true,
+        "name": "schema",
+        "options": [],
+        "query": {
+          "query": "label_values({__name__=~\"pg_up|mysql_up|mongodb_up|proxysql_mysql_status_active_transactions\"}, schema)",
+          "refId": "Metrics-schema-Variable-Query"
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 5,
+        "tagValuesQuery": "",
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "current": {},
+        "datasource": "Metrics",
+        "definition": "label_values(pg_up{service_name=\"$service_name\"}, service_id)",
+        "hide": 2,
+        "includeAll": false,
+        "label": "Service ID",
+        "multi": false,
+        "name": "service_id",
+        "options": [],
+        "query": {
+          "query": "label_values(pg_up{service_name=\"$service_name\"}, service_id)",
+          "refId": "Metrics-service_id-Variable-Query"
+        },
+        "refresh": 2,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 5,
+        "tagValuesQuery": "",
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "current": {},
+        "datasource": "Metrics",
+        "definition": "label_values(patroni_version{environment=~\"$environment\",cluster=~\"$cluster\",service_name=\"$service_name\"}, scope)",
+        "hide": 0,
+        "includeAll": false,
+        "label": "Scope Name",
+        "multi": false,
+        "name": "scope_name",
+        "options": [],
+        "query": {
+          "query": "label_values(patroni_version{environment=~\"$environment\",cluster=~\"$cluster\",service_name=\"$service_name\"}, scope)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 2,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 5,
+        "tagValuesQuery": "",
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      }
+    ]
+  },
+  "time": {
+    "from": "now-3h",
+    "to": "now"
+  },
+  "timepicker": {
+    "hidden": false,
+    "refresh_intervals": [
+      "5s",
+      "10s",
+      "30s",
+      "1m",
+      "5m",
+      "15m",
+      "30m",
+      "1h",
+      "2h",
+      "1d"
+    ],
+    "time_options": [
+      "5m",
+      "15m",
+      "1h",
+      "6h",
+      "12h",
+      "24h",
+      "2d",
+      "7d",
+      "30d"
+    ]
+  },
+  "timezone": "",
+  "title": "Patroni Details",
+  "uid": "rLzu8z_Vk",
```

```suggestion
  "uid": null,
```
FYI: we should not commit the IDs - for provisioned dashboards those get generated by grafana at the moment of provisioning.